CHANGELOG.md

```diff
@@ -3,6 +3,176 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0
+
+_Mar 20, 2024_
+
+We're excited to [announce the first v7 stable release](https://mui.com/blog/mui-x-v7/)! 🎉🚀
```

Comment:
The blog is not live yet 🤔 Should we include the link in that case?

Reply:
I think it's OK.
We can follow this order:

- Do the release (without posting to GitHub Releases)
- Publish the blog post
- Post to GitHub Releases

What do you think?

Reply:
Or we can release without a link in both the readme as well as the release post and add it in when the blog post is live. 🤔

Reply:
If we wait for the blog post, then we should be super confident that it will be on time and not delayed (which occurs in the past)

Reply:
In that case, it's probably safer to remove the link for now and circle back when the blog post is live 👌

Reply:
> If we wait for the blog post, then we should be super confident that it will be on time and not delayed (which occurs in the past)

I'd say the blogpost is already ready for deploy, if that's where the uncertainty lies.

Reply:
Okay 👌 
Just to double check: we are sure that the blog post will be released early next week? And we are all okay with leaving the changelog as it is and posting to GitHub releases only after the core release with the blog post?

Reply:
If we plan to circle back and edit the file after the blogpost is live, I think we could link to the blog post preview: https://deploy-preview-41563--material-ui.netlify.app/blog/mui-x-v7/

Reply:
```suggestion
We're excited to [announce the first v7 stable release](https://deploy-preview-41563--material-ui.netlify.app/blog/mui-x-v7/)! 🎉🚀
```

Reply:
> we are sure that the blog post will be released early next week?

Why waiting till the next week? I thought we would release everything today, including the blog post 🙂